drivers/modem/gsm_ppp.c

```diff
@@ -681,5 +694,32 @@ static int gsm_init(struct device *device)
 	return 0;
 }
 
+void restart_gsm_init(void)
+{
+	int r;
+
+	is_booting_phase = false;
+
+	/*
+	 * Switching to modem UART interface to enable the modem for CONNECT
+	 */
+
+	r = modem_iface_uart_init_dev(&gsm.context.iface,
+			CONFIG_MODEM_GSM_UART_NAME);
+
```

Comment:
We are not using the return value for anything. If this is intended, the you can just write this as `(void)modem_iface_uart_init_dev(...);`, otherwise add some check here. Also the indentation is wrong for the config option line.

Reply:
addressed, may be indentation needs improvement